src/Sylius/Bundle/CoreBundle/Validator/Constraints/OrderPaymentMethodPerChannelEligibility.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+
+final class OrderPaymentMethodPerChannelEligibility extends Constraint
```

Comment:
```suggestion
final class OrderPaymentMethodEnabledInChannel extends Constraint
```
Maybe something like that here in other places 🤔

Reply:
or rather OrderPaymentMethodEnabledInChannelEligibility ? And in corresponding places to

Reply:
Changed as suggested